### ISSUE TITLE
After running the program, the performance of the model decreases.

### ISSUE THREAD

**Issue by yxsysu** [Simon4Yan/eSPGAN#10]
I ran the code from 'https://github.com/Simon4Yan/Person_reID_baseline_pytorch' using training data on Duke and obtain a model with 48.8% rank1 accuracy on Market. But after running espgan.py with the default parameters, the model only achieved 41.0 rank1 accuracy on Market...
I want to know what may have gone wrong. Thanks a lot!

**Comment by yxsysu** [comment]
I read the paper, and I found that the problem may come from here: 'Specifically, we normalize the image with the same mean (0.5, 0.5, 0.5) and standard deviation (0.5, 0.5, 0.5) for both the image translator and the feature learner' while the mean and std used for normalization in 'https://github.com/Simon4Yan/Person_reID_baseline_pytorch'  is [0.485, 0.456, 0.406], [0.229, 0.224, 0.225].
